modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/creating-first-rosa-cluster.adoc
+
+
+[id="rosa-accessing-your-cluster-quick"]
+= Accessing your cluster
+
+To log in to your cluster, you can use this quick access procedure.
+
+
+[NOTE]
+====
+As a best practice, xref:../getting_started_rosa/accessing-rosa-cluster.html#rosa-accessing-your-cluster[access your cluster with an IDP account] instead.
+====
+
+
+.Procedure
+
+. Enter the following command:
++
+[source, terminal]
+----
+$ rosa create admin -c <cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
+I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
+I: To login, run the following command:
+oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \
+--username cluster-admin \
+--password FWGYL-2mkJI-3ZTTZ-rINns
+~/g/s/g/openshift (⎈ ocm-name/api-crc-testing:6443/kube:admin) $    oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \
+>  --username cluster-admin \
+>  --password FWGYL-2mkJI-3ZTTZ-rINns
```

Comment:
Is this part of the command output or a different command?

Reply:
Good catch - the video demo went fast. Upon reading it, it should be its own step in the procedure with example output. Done.